src/CONST.ts

```diff
@@ -596,6 +597,10 @@ const CONST = {
             },
             THREAD_DISABLED: ['CREATED'],
         },
+        RESOLUTIONS: {
```

Comment:
Sorry can we rename this to be a bit more specific? I missed this earlier.

```suggestion
        ACTIONABLE_MENTION_WHISPER_RESOLUTION: 
```